client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -171,6 +177,9 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 	} = useContext( WCPaySettingsContext );
 
 	const onModalClose = () => {
+		if ( isLoading ) {
+			return;
+		}
```

Comment:
More renaming suggestions 😁 

`onModalClose` reads strange to me – what is its purpose? Currently the name is based on where it is bound (a kind of coupling?). With the addition of the early return, it's becoming abstract, e.g. `onModalClose() { if isLoading don't close }`.

Reply:
I've renamed it to `handleModalClose` in 27e7621 – does this make more sense?

Reply:
A tiny bit!